flow/Platform.cpp

```diff
@@ -2787,7 +2787,8 @@ void crashHandler(int sig) {
 	fprintf(stderr, "SIGNAL: %s (%d)\n", strsignal(sig), sig);
 	fprintf(stderr, "Trace: %s\n", backtrace.c_str());
 
-	_exit(128 + sig);
+	signal(sig, SIG_DFL);
```

Comment:
It appears the use of `signal` is discouraged and `sigaction` is preferred (see e.g. http://man7.org/linux/man-pages/man2/signal.2.html). Not sure if the problems with it have any relevance here, but if we can use `sigaction` instead, we might as well.

Reply:
In the event of an error in this function (or `sigaction`), I'm not sure what will happen with the subsequent `kill`. Not sure if we can actually get errors here, but maybe it's wise to have a safety mechanism to `_exit` just in case we do?

Reply:
I now use sigaction, and call _exit if any of sigemptyset, sigaction, or kill fail. PTAL